### DISCUSSION TITLE
Where can I add an htaccess file to an airflow docker compose image webserver

### DISCUSSION THREAD

**Discussion by copypasteearth** [apache/airflow#33489] [General]
Hi everyone, i am looking for where to add some custom htaccess rules and things in the webserver part of an airflow docker compose server. Does anyone know how to edit the htaccess file of airflow?

**Comment by potiuk** [comment]
This is a generic python FLASK server that airflow runs, htaccess is an apache webserver concept - and if you need to use htaccess you need add apache webserver as proxy in front of airflow webserver.
